apollo-router/src/plugins/cache/invalidation.rs

```diff
@@ -230,7 +230,7 @@ pub(crate) enum InvalidationRequest {
 }
 
 impl InvalidationRequest {
-    fn key_prefix(&self) -> String {
+    fn key_prefix(&mut self) -> String {
```

Comment:
I think this requires a comment to explain why it's mut. OR, maybe even better, sort when the InvalidationRequest is constructed?
```suggestion
    /// Compute a cache key prefix. For entity keys, this destructively sorts all objects.
    fn key_prefix(&mut self) -> String {
```